src/components/JsPsychExperiment.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Experiment, jsPsych } from "jspsych-react";
+import { tl } from "../timelines/main";
+import "../App.css";
+import "bootstrap/dist/css/bootstrap.css";
+import "@fortawesome/fontawesome-free/css/all.css";
+import { rt_categorize_html } from "../lib/rt-categorize-html";
+
+function JsPsychExperiment({ dataUpdateFunction, dataFinishFunction }) {
+  console.log("Outside Turk:", jsPsych.turk.turkInfo().outsideTurk);
```

Comment:
remove console logging, should the css imports be at the app level? otherwise, the component here looks great!